### PR TITLE
Prepare PROW for hostpath provisioner

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

### CODE REVIEW

**Comment by awels** [comment]
@stu-gott @rmohr I am trying to add some CI and functional tests to the hostpath provisioner. I am following the example from CDI.

File: github/ci/prow/files/jobs/hostpath-provisioner/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+filters:
+  ".*":
+    reviewers:
+      - awels
```
Comment (rmohr) [inline github/ci/prow/files/jobs/hostpath-provisioner/OWNERS]: Best to add someone else  from your team too. Otherweise you will either have to configure prow for your repo, that authors of their PR can also /lgtm their own code, or you will have to rely on external reviewers too.

Comment (awels) [reply github/ci/prow/files/jobs/hostpath-provisioner/OWNERS]: Sure, I can add some people.

**Comment by awels** [comment]
@rmohr Since hostpath provisioner is already in kubevirt org, it should just work right?

**Comment by rmohr** [comment]
Could be that we forgot to set something, but in general yes.

**Pull request merged**